src/plugins/changelog/theme/ChangelogItem/Header/index.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
```

Comment:
We need to correct this. Correct this language by adding the `LICENSE` to the `ChangelogItem/` folder. See https://github.com/grafana/grafana/blob/main/NOTICE.md for example.

Reply:
Update this file and related files.